playgrounds/vite/tsconfig.json

```diff
@@ -9,8 +9,8 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "ESNext",
-    "moduleResolution": "Node",
+    "module": "Node16",
+    "moduleResolution": "Node16",
```

Comment:
https://github.com/iTwin/iTwinUI/pull/1549/commits/efb929b1139205be700a7ade69bd63668ab9afdd: Change `moduleResolution` for vite and next to "bundler". That also enabled me to undo the workaround (https://github.com/iTwin/iTwinUI/pull/1549#discussion_r1318791764) and also undo some other changes that were needed when `"moduleResolution": "Node16"`.

I noticed that running `yarn build` in next-playground gave the below message. But since the build passed, I guess it is alright.

![image](https://github.com/iTwin/iTwinUI/assets/45748283/3af9e2b4-07d4-4ec7-a475-5def1ff15b74)

<!-- ![image](https://github.com/iTwin/iTwinUI/assets/45748283/f17ed59f-3eec-4751-bec0-75a52a3d1a59) -->

Reply:
when using bundler mode, you can import the real file extensions (`.tsx`)

and i don't think nextjs support `bundler`, so i would continue using `nodenext` or `node16` there

Reply:
> when using bundler mode, you can import the real file extensions (`.tsx`)

To confirm, are you suggesting changing something? Because as of now, the vite-playground (that uses bundler) build passes without changing anything.

> and i don't think nextjs support `bundler`, so i would continue using `nodenext` or `node16` there

https://github.com/iTwin/iTwinUI/pull/1549/commits/a22d11c74b5fb2a449ab4b52d3889acf2d5d10e4: Sounds good, reverted back to using `Node16`

Reply:
yes, i'm suggesting changing local imports inside the vite playground to use `.tsx` file extension

```diff
- import App from './App';
+ import App from './App.tsx';
```

Reply:
https://github.com/iTwin/iTwinUI/pull/1549/commits/79602768c0d7e1e55ce7e7370967696efba09a70: I found only one local/relative import, which is the one that you shared above. I made that change. I also added `"allowImportingTsExtensions": true` to the tsconfig to make this work.